flow360/component/simulation/outputs/outputs.py

```diff
@@ -0,0 +1,126 @@
+from typing import List, Optional, Tuple, Union
+
+import pydantic as pd
+
+from flow360.component.simulation.framework.base_model import Flow360BaseModel
+from flow360.component.simulation.framework.entity_base import EntityList
+from flow360.component.simulation.outputs.output_entities import (
+    Isosurface,
+    ProbeMonitor,
+    Slice,
+    SurfaceIntegralMonitor,
+)
+from flow360.component.simulation.primitives import Surface
+from flow360.component.simulation.types import NonNegativeAndNegOneInt
+
+"""Mostly the same as Flow360Param counterparts.
+Caveats:
+1. Check if we support non-average and average output specified at the same time in solver. (Yes but they share the same output_fields)
+2. We do not support mulitple output frequencies for the same type of output.
+"""
+
+# TODO: We need a proper list
+SurfaceOutputFields = List[str]
+VolumeOutputFields = List[str]
+SliceOutputFields = List[str]
+MonitorOutputFields = List[str]
```

Comment:
can we use this: https://github.com/flexcompute/Flow360/blob/develop/flow360/component/flow360_params/flow360_fields.py
I think this good, pydantic agnostic design

Reply:
Sure